### PR TITLE
Get Custom Locations RP app by object id

### PR DESCRIPTION
**Pull request by TheOnlyWei**
ignore error if cannot get custom locations app by object id
update to major version 2.0.0 for enabled-features security update breaking change

---

### Related command
az connectedk8s

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?
Can't run it, seems to be a bug with the script as I get the below error:
```
Traceback (most recent call last):
  File "C:\tasks\TheOnlyWei\azure-cli-extensions\scripts\ci\test_index.py", line 23, in <module>
    from wheel.install import WHEEL_INFO_RE
ModuleNotFoundError: No module named 'wheel.install'
```

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/connectedk8s/setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '1.5.0'
+VERSION = '2.0.0'
```
Comment (zhoxing-ms) [inline src/connectedk8s/setup.py:20]: We usually only upgrade the major version when there are some breaking changes in PR. Therefore, it is recommend to upgrade the secondary version for this case

Comment (zhoxing-ms) [inline src/connectedk8s/setup.py:20]:
```suggestion
VERSION = '1.6.0'
```

Comment (TheOnlyWei) [reply src/connectedk8s/setup.py:20]: This is to compensate for the previous breaking change, which still has a breaking change. I initially upgraded only the minor version, due to some assumptions from the code, but those were wrong. So, I updated the major version to make it clear to customers that there is a breaking change.

**Pull request closed without merging**